backend/src/main/scala/com/softwaremill/bootzooka/email/EmailService.scala

```diff
@@ -40,19 +41,19 @@ class EmailService(emailModel: EmailModel, idGenerator: IdGenerator, emailSender
       emailModel.count().transact(xa).map(_.toDouble).map(Metrics.emailQueueGauge.set)
     }
 
-    Task.parZip2(sendProcess, monitoringProcess)
+    sendProcess.flatMap(r => monitoringProcess.flatMap(d => IO(r, d)))
```

Comment:
that will work differenlty. Previously, we were runing send & monitor *in parallel*. Now, they would run sequentially - causing monitoring never to run, as send never finishes. So we need to change this back to parallel